packages/core/src/internal/esm/cjsBindings.ts

```diff
@@ -0,0 +1,8 @@
+import type { Exports } from "cjs-module-lexer";
+import { init, parse as cjsParse } from "cjs-module-lexer";
+
+await init();
```

Comment:
Hmm... I guess this is the best option for the moment, but I’ll probably make a breaking change at some point to make the analysis itself async instead of having TLA.

Reply:
Will TLA work with your current web deployment? I hadn't taken a look at that.

They also include a pure-JS implementation which does not need to initialize WASM: https://github.com/nodejs/cjs-module-lexer/blob/main/lexer.js. Strangely, it is only available via CommonJS. Via `import` you always get the WASM version.

An alternative, rather than drilling `async` into the entire resolution invocation graph, would be to surface this as `init` on `@arethetypeswrong/core`.

Reply:
I _think_ TLA should be ok with the current Vite setup. `npm start` at the repo root launches the dev server, and `npm build` will include the prod web build.